.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,7 +24,9 @@
 <!-- Go over all points below, and after creating the PR, tick the checkboxes that apply. -->
 <!-- If you're unsure about any of these, don't hesitate to ask in the Help Contributors room linked above. We're here to help! -->
 - [ ] Tested changes locally.
-- [ ] Closes currently open issue (replace XXXX with an issue no): Closes #XXXX
+- [ ] Addressed currently open issue (replace XXXXX with an issue no in next line) 
+
```

Comment:
If you remove this empty line, will it still work? Without that empty line, it's grouped together a bit better.

Reply:
Yup, it should work, but I believe the visual spacing draws attention to the keyword much better, and contributors are likely to notice more effectively.